### DISCUSSION TITLE
Tailwind Elements Stable v1.0.0. - a free, open-source UI Kit with 500+ components integrated with Svelte - is out.

### DISCUSSION THREAD

**Discussion by MaciejZimoch** [sveltejs/kit#10829] [Show and tell]
Tailwind Elements Stable v1.0.0. - a free, open-source UI Kit with 500+ components integrated with Svelte - is out.
And we couldn't be prouder, so give it a go [here](https://tailwind-elements.com/docs/standard/integrations/svelte-integration) :)

It will change Your life (or at least how smooth & painless web development is) ;)

It's stable & reliable, comes with over 500 UI Components, easy theming & customisation, dark mode, Figma design system AND free hosting :) .

If You know something about JS, You probably have heard about Svelte.js. And If You know anything about CSS, You have for sure heard about Tailwind Elements. Why not work with them both, when they're integrated so splendidly?

For all JS aficionados: By default all components have autoinit which means they are initialised by data attributes. But if you want to make init by JS - there is also possibility to do that.

Check out how to do it and more about Tailwind Elements [here](https://tailwind-elements.com/docs/standard/integrations/svelte-integration)

Even though TE is already a fully functional UI Kit, it's still in its infancy really - we have great plans and a long and exciting road ahead of us. So whatever input (however critical) You might have, please share it with us here or on any of our channels on which we will be more than happy to host you:
[Discord ](https://discord.com/invite/ZkTVKaJC)
[GitHub](https://github.com/mdbootstrap/Tailwind-Elements) 
[X ](https://twitter.com/TailwindElement)

**Comment by stalkerg** [comment]
It's not a native integration, it's basicaly your JS on top of generated DOM by Svelte. I expected a tons of issues during reactivities.
